other/deny-default-service-account-bindings/deny-default-service-account-bindings.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-default-service-account-bindings
+  annotations:
+    policies.kyverno.io/title: Deny binding of default service accounts
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.11.0
+    policies.kyverno.io/minversion: 1.10.0
+    kyverno.io/kubernetes-version: "1.26"
+    policies.kyverno.io/description: >-
+      The default service account is automatically mounted into all pods in a namespace unless 
+      explicitly overridden. If this account is bound to a Role or ClusterRole that grants extensive 
+      permissions, every pod in the namespace using the default service account will inherit these 
+      permissions. This setup can lead to unnecessary security risks if a pod is compromised, as an 
+      attacker could potentially gain access to other resources within the cluster.For an enhnaced 
+      security, using the default service account in RoleBindings is not recommended.
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - name: deny-default-service-account-bindings
+      match:
+        resources:
+          kinds:
+            - RoleBinding
+            - ClusterRoleBinding
```

Comment:
Rules must use `any` or `all` under the `match` and `exclude` blocks to be conformant to the current schema.